framework/doc/content/source/correctors/PointwiseRenormalizeVector.md

```diff
@@ -0,0 +1,21 @@
+# PointwiseRenormalizeVector
+
+!syntax description /Correctors/PointwiseRenormalizeVector
+
+This user object can pointwise renormalize the solution for a set of variables, taking each variable as the component of a vector and scaling the variables to obtain the user specified L2-norm.
```

Comment:
Not a user object anymore

Reply:
I have a related comment: I understand that right now correctors are basically userobjects in the backend. But should we still advertise that fact? Not just in this documentation markdown, I also saw similar emphasis in docstrings elsewhere. For example, the postprocessors also derive from userobject, but we don't explicitly emphasize that fact anywhere. Likewise, I feel we probably don't need to mention correctors being userobjects from times to times. What do you think?

Reply:
I think it's fine. If users can't understand inheritance between Correctors and User Objects they won't like MOOSE much anyway. 
And developers can always use the reminder since the execution scheduling of Correctors is tied with the user objects, same as the postprocessors. So they do need to know

Reply:
Yeah, I think we don't require users to have OOP background, so maybe we won't expect them to understand the specific word "inheritance", but I agree we can communicate to users that "correctors are user objects behind the scenes".  So I would suggest explaining this bit in `Correctors/index.md`.